src/FetchMockServer.ts

```diff
@@ -1,55 +1,61 @@
-import { BaseServer, type BaseServerOptions } from './BaseServer.js';
```

Comment:
Could you move the adapters into an "adapters" directory? The current dir is getting hard to read

Reply:
Done